ocs/ocs_agent.py

```diff
@@ -1372,7 +1374,11 @@ def get(self, key, default=ParamError(''), check=None, cast=None, type=None,
 
         """
         self._checked.add(key)
-        value = self._params.get(key, None)
+        if self._params is None and not isinstance(default, ParamError):
+            self._params = {}  # avoid attribute error in check_for_strays
+            value = default
+        else:
+            value = self._params.get(key, None)
```

Comment:
Is it perhaps simpler to detect and convert None -> {} in the `__init__`?

Reply:
Mhm, when I considered that I dropped the `self._params is None` in line 1377, but then detecting when to use the default value seemed tricky. But I didn't realize at the time you were handling that in the lines below...

New implementation in [1a4257c](https://github.com/simonsobs/ocs/pull/333/commits/1a4257c89254b1826bc56d1da18f1f8456db409e).